compiler/noirc_driver/src/lib.rs

```diff
@@ -139,6 +139,10 @@ pub struct CompileOptions {
     /// A lower value keeps the original program if it was smaller, even if it has more jumps.
     #[arg(long, hide = true, allow_hyphen_values = true)]
     pub max_bytecode_increase_percent: Option<i32>,
+
+    /// Flag to skip tests using oracles.
+    #[arg(long, hide = true)]
+    pub skip_oracle: bool,
```

Comment:
Shouldn't this be added to the [TestCommand](https://github.com/noir-lang/noir/blob/d7a3207b6c3c1c487ce29282730b0ba44481410c/tooling/nargo_cli/src/cli/test_cmd.rs#L26) instead? It sounds more like a testing option than a general compile option which is shared with e.g. `execute` and `info`.

Reply:
Done